### ISSUE TITLE
Selenium tests do not run against node v0.10.x

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/persona#3895]
This occurs after blowing away the `node_modules` directory and re-installing. 

```
➜  automation-tests git:(dev) ✗ ./scripts/run-all.js -p 1 -e dev
adding add-primary-to-primary.js
adding add-primary-to-secondary.js
adding oncancel.js
adding cancel-account.js
adding change-password-test.js
adding frontend-qunit-test.js
adding health-check-tests.js
adding broken-primary.js
adding primary-shuts-down-single-email.js
adding primary-starts-up-single-email.js
adding transition-to-secondary-forgot-password.js
adding transition-to-secondary.js
adding new-user-primary-test.js
adding new-user-secondary-test.js
ignoring public-terminals.js
adding remove-email.js
adding reset-password-test.js
adding returning-user.js
adding sign-in-test.js
Running 18 suite(s) on 1 platform(s), 1 at a time against https://login.dev.anosrep.org
><><><><><><><><><><><><><><><><><><

18/18 tests passed
➜  automation-tests git:(dev) ✗
```

**Comment by jrgm** [comment]
A fix for where it crashes (lib/vows/suite.js:230) was done here 
https://github.com/cloudhead/vows/commit/f412e4a568481817faea0ce5c8f2bc53f6153f67

Trying vows@0.7.0 now, which is having more luck.

**Comment by jrgm** [comment]
Anyways, current status is that with this diff, things work. 

```
diff --git a/automation-tests/package.json b/automation-tests/package.json
index cd58c91..b9b8f0b 100644
--- a/automation-tests/package.json
+++ b/automation-tests/package.json
@@ -4,7 +4,7 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
-    "vows": "0.6.4",
+    "vows": "git://github.com/cloudhead/vows#253ca34c",
     "wd": "git://github.com/lloyd/wd#bb612777",
     "request": "2.12.0",
     "optimist": "0.3.5",
```

But there's a separate issue, which is that the http-proxy is leaking sockets until it won't accept any more connections, causing these tests to eventually fail (unless you `sudo su - proxy; forever restartall`.)

I filed a different bug in a bit, when I narrow this down.

**Comment by jrgm** [comment]
Um, so the reason I'm pulling the SHA, is that the current release version of vows on npm is 0.7.0, which fixes the problem of running on node 0.10.0. 

**But**, @shane-tomlinson ... remember that problem about vows not running test suites beginning with a 'C'. Well, it got re-introduced in vows@0.7.0. The first checkin after 0.7.0 fixes it again (and puts a test in (again?)). 

Looking at the changes up too HEAD, and the rate of changes after the 'not C:\ drive' fix, it seemed reasonable to just move to HEAD, which is `vows#253ca34c`.

**Comment by shane-tomlinson** [comment]
Thanks for tracking this down @jrgm.
